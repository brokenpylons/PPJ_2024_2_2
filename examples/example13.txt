var n = 5;
var result = 1;
for (var i = 1, n) {
    result = result * i;
}

print(result);
