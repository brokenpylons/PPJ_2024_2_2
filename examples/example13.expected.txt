define("var") variable("n") assign("=") real("5") term(";") define("var") variable("result") assign("=") real("1") term(";") for("for") lparen("(") define("var") variable("i") assign("=") real("1") to(",") variable("n") rparen(")") begin("{") variable("result") assign("=") variable("result") times("*") variable("i") term(";") end("}") print("print") lparen("(") variable("result") rparen(")") term(";") 
